modules/python/src2/cv2.cpp

```diff
@@ -290,6 +335,64 @@ bool parseNumpyScalar(PyObject* obj, T& value)
     return false;
 }
 
+#ifdef CV_CXX11
+thread_local
+#endif
+std::vector<std::string> conversionErrors;
+
+void pyRaiseCVOverloadException(const std::string& functionName)
+{
+    const std::size_t conversionErrorsCount = conversionErrors.size();
+
+    // In modern std libraries small string optimisation is used = no dynamic memory allocations
+    // but it can be applied only for string with length < 18 symbols (in GCC)
+    const char* header = "Overload resolution failed:";
+    const std::size_t header_size = 27;
+    const std::string bullet = "\n - ";
+
+    // Estimate required buffer size - save dynamic memory allocations = faster
```

Comment:
> Raise

This code works in case of errors only, so it is not a part of code flow critical path (no need to optimize this aggressively).

Reply:
True, but it is ~x8 times [faster](https://quick-bench.com/q/__TRRxUezkhEpBqxTWPmz3r-J_A) than `std::ostringstream` variant... :nerd_face: